Deploy/include targets

Wenzel opened the pull request:
This PR include the kafl.targets repo by default when installing the fuzzer
Also rename kafl.targets to `examples`

il-steffen commented:
Are you renaming the destination folder kafl/targets as well? What about the repo?

Wenzel commented:
@il-steffen good catch, I renamed the destination folder from `targets` to `examples`.
Also added a new env var in `env.sh`: `EXAMPLES_ROOT`

Is it necessary to rename the kafl.targets repository ? Or that would be confusing, and it doesn't change anything for people deploying kAFL through the playbook anyway ?
